content/applications/finance/expenses/post_expenses.rst

```diff
@@ -0,0 +1,74 @@
+=============
+Post expenses
+=============
+
+Once an expense report is :ref:`approved <expenses/approve>`, the next step is to post the expense
+report to the proper accounting journal.
+
+.. important::
+   To post expense reports to an accounting journal, the user **must** have the following
+   :doc:`access rights <../../general/users/access_rights>`:
+
+   - Accounting: Accountant or Adviser
+   - Expenses: Manager
+
+Only expense reports with an :guilabel:`Approved` status can post the expenses to a journal. To view
```

Comment:
```suggestion
Only expense reports with an *Approved* status can post the expenses to a journal. To view
```
opting for italics here because, at this point in the doc, the reader/user is **not** seeing anything (in particular) on their screen. In the next sentence, you instruct them to the correct reporting page, where that *does* become present. But, since this first sentence takes place prior to that instruction, italics are used.